daemon/daemon.go

```diff
@@ -743,6 +743,10 @@ func NewDaemon(config *Config, registryService *registry.Service) (daemon *Daemo
 	}
 
 	sysInfo := sysinfo.New(false)
+	if !sysInfo.CgroupDevicesEnabled {
+		return nil, fmt.Errorf("Error mounting cgroup devices")
```

Comment:
Error is pretty weird. Should be like `Devices cgroup isn't mounted` or something like this. Also makes sense to write why it's bad.